src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentation.cs

```diff
@@ -29,11 +29,10 @@ internal class AspNetCoreInstrumentation : IDisposable
         /// <summary>
         /// Initializes a new instance of the <see cref="AspNetCoreInstrumentation"/> class.
         /// </summary>
-        /// <param name="activitySource">ActivitySource adapter instance.</param>
         /// <param name="options">Configuration options for ASP.NET Core instrumentation.</param>
-        public AspNetCoreInstrumentation(ActivitySourceAdapter activitySource, AspNetCoreInstrumentationOptions options)
+        public AspNetCoreInstrumentation(AspNetCoreInstrumentationOptions options)
```

Comment:
I initially thought (inccoreclty) this is a breaking change! The public api analyzer is a gift!

Reply:
Should we change this to internal?